.idea/compiler.xml

```diff
@@ -64,4 +78,4 @@
       <module name="fdb-record-layer_test" target="1.8" />
     </bytecodeTargetLevel>
   </component>
-</project>
+</project>
```

Comment:
Could you remove the changes to this file? They don't look wrong to me, but I don't think we want to couple these together.